smart_campus/app/management/commands/load_beacon_data.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.gis.geos import Point
+
+import pandas as pd
+
+from app.models import Beacon, UserGroup
+
+
+class Command(BaseCommand):
+    help = 'Insert beacon datas from xls file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('path', nargs=1, type=str)
+
+    def handle(self, *args, **options):
+        try:
+            data = pd.read_excel(options['path'][0])
+            for i in range(0, len(data)):
```

Comment:
Use `for index, datum in enumerate(data)` instead.

Reply:
enumerate(data) only shows column names
I'll use iterrows with pandas.DataFrame